contrib/kube-prometheus/README.md

```diff
@@ -101,7 +101,11 @@ In order to update the kube-prometheus dependency, simply use the jsonnet-bundle
 
 ### Compiling
 
-e.g. of how to compile the manifests: `./build.sh example.jsonnet`
+Compiling is done with [mixtool](https://github.com/metalmatze/mixtool) which bundles [go-jsonnet](https://github.com/google/go-jsonnet) and adds some more convenience helpers like converting to YAML and linting the output.
+
+> `mixtool` can be installed with `go get -u github.com/metalmatze/mixtool/cmd/mixtool`
+
+Example of how to compile the manifests: `mixtool build -m manifests/ example.jsonnet`
```

Comment:
<3

Reply:
maybe we should default to outputting json and make yaml output an argument

Reply:
`--yaml` is already a flag. It is simply `true` by default. We can easily flip it. Though I think that most people want YAML. Maybe we should add a `--json` flag which is false by default instead. Might make more sense from a UX perspective.

Reply:
I think we need to be careful not to mix kubernetes and mixtool concerns. mixtool really should just be optimized for mixins.

Reply:
Yep, totally. But I still think that most people want to output YAML these days.
Moving the discussion to the mixtool itself: https://github.com/metalmatze/mixtool/issues/2

Reply:
Should we merge this PR nevertheless? WDYT?

Reply:
as long as mixtool is a moving target in terms of what it renders, I'd rather not as that could cause builds to be non-reproducible if mixtool master changes